gribscan/gribscan.py

```diff
@@ -344,6 +343,13 @@ def scan_gribfile(filelike, **kwargs):
             **kwargs,
         }
 
+        if (param := m.get("shortName")) != "unknown":
```

Comment:
Is there a possibility, that no `shortName` key exists at all? Anf if so, would we want to do the same as in the "unknown" case? E.g.:

```suggestion
        if (param := m.get("shortName", "unknown")) != "unknown":
```

Reply:
That's a good question... I would assume that ecCodes will always attempt to return a `shortName`, so "unknown" is the "worst" case. But we can implement your safeguard anyhow 👍